src/mrmsg.h

```diff
@@ -26,6 +26,10 @@
 extern "C" {
 #endif
 
+#ifdef PY_CFFI
+typedef int... time_t;
+#endif
```

Comment:
Move external declarations to `_build.py`, no need to pollute the actual headers with this CFFI detail.

Reply:
well, if time.h is missing for CFFI it may be also missing for other systems. So, i think it is fine just to include <time.h>. But you are right, it's only a detail.

Reply:
I think you misunderstand.  The code does and should include time.h.  However this header file is a bit complicated for the cffi parser so I'd guard the import -together with all other external headers- with an #ifdef PY_CFFI and thus not include it for cffi.  However cffi still needs to know that time_t is some kind of integer, which is what this "declaration" does (cffi then uses the compiler to work out exactly what size this integer is etc).  Hence this cffi-specific declaration of time_t should be moved to _build.py.

Reply:
okay, I see :) - and a happy new year :)